robot_calibration/include/robot_calibration/capture/robot_finder.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2022 Michael Ferguson
+ * Copyright (C) 2014-2017 Fetch Robotics Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
can you add a brief description of the robot_finder's purpose?

Reply:
I'm planning to update the README once I have an example to point at (which I need to cleanup and merge to the UBR-1 repo)